### ISSUE TITLE
runit could use a more general explaination

### ISSUE THREAD

**Issue by bobertlo** [void-linux/void-docs#122]
@Duncaen mentioned this in irc and I think it is a good idea. Right now we have a short quick usage guide in the header of the `Configuration > Services and Daemons` section. I think a possible layout could be:

```
- Configuration
   - Runit
      - Starting/Stopping Services
      - Configuring Services
   - Services and Daemons
      - Cron
      - ...
```

With an explaination of what runit is and why it is used in the heading, but I am very open to discussion on that point. There's probably a third and better solution?

Especially in the context of people viewing Void Linux as an "anti-systemd" distro, I think we could mention how we are more "pro-runit" and dropped systemd promarily over compatibility issues.

**Comment by bobertlo** [comment]
Now that I look at the docs again, I think a good writeup about runit might be better suited to go in the initial `About` section.

**Comment by flexibeast** [comment]
i can do the 'Runit' subsection of 'Configuration'; i assume the more general writeup about why Void is pro-Runit would be best left for someone on the Void dev team.

**Comment by flexibeast** [comment]
So should i simply port [the 'Runit' page on the wiki](https://wiki.voidlinux.org/Runit)?

**Comment by bobertlo** [comment]
I just took a quick look at the wiki page. That seems like a good start. I do generally like the layout of the wiki page more, but some things could be organized better. I'd really like if that information could be split into at least a couple pages. Maybe something like:

- runit
  - Basic Usage
  - Creating/Modifying Services

Or there could be `managing services` and `configuring services`?

Basically it looks like if we get everything from both the docs and the wiki merged into a more organized layout we're in good shape. I'm just brainstorming.

**Comment by flexibeast** [comment]
So the stuff currently on [the "Services and daemons" page of the handbook](https://docs.voidlinux.org/config/services/index.html) would be removed, with that information instead being available via "Basic Usage" and "Creating/Modifying Services" pages under a "runit" (or maybe "runit: services and daemons"?) subsection of the handbook? Into which we could add anything from the wiki not already included?

**Comment by bobertlo** [comment]
Right, that is what I was thinking. `Services and daemons` would stay, but for clarity would be a separate section from runit.

**Comment by flexibeast** [comment]
With #127 now merged, should this issue be closed, or should it remain open until

> we could mention how we are more "pro-runit" and dropped systemd promarily over compatibility issues

has been addressed?

**Comment by bobertlo** [comment]
I think #131 should take care of the rest of this.
